environment2.yml

```diff
@@ -0,0 +1,51 @@
+name: simple-db
+channels:
+  - defaults
+dependencies:
+  - ca-certificates=2020.10.14=0
+  - certifi=2020.11.8=py39hecd8cb5_0
+  - libcxx=10.0.0=1
+  - libedit=3.1.20191231=h1de35cc_1
+  - libffi=3.3=hb1e8313_2
+  - ncurses=6.2=h0a44026_1
+  - openssl=1.1.1h=haf1e3a3_0
+  - pip=20.2.4=py39hecd8cb5_0
+  - python=3.9.0=h88f2d9e_2
+  - readline=8.0=h1de35cc_0
+  - setuptools=50.3.1=py39hecd8cb5_1
+  - sqlite=3.33.0=hffcf06c_0
+  - tk=8.6.10=hb0a8c7a_0
+  - tzdata=2020d=h14c3975_0
+  - wheel=0.35.1=pyhd3eb1b0_0
+  - xz=5.2.5=h1de35cc_0
+  - zlib=1.2.11=h1de35cc_3
+  - pip:
+    - asdf==2.7.1
+    - astrodbkit2==0.2.1.dev7+g1909568
```

Comment:
I think this will fail since nothing here specifies that pip should use the github version of this package. 
In the long run (and for stability), I think it's OK that users just do pip install astrodbkit2 and not go the github route. 
We can do that since we're still in active development.

Reply:
so should I delete this from the environment file?